tdp_core/db.py

```diff
@@ -137,9 +141,7 @@ def run(self, sql, **kwargs):
     :param kwargs: args for this query
     :return: list of dicts
     """
-    parsed = to_query(sql, self._supports_array_parameter, kwargs)
-    _log.info('%s (%s)', parsed, kwargs)
```

Comment:
@dvvanessastoiber: Why was this removed?

Reply:
Because this code is duplicated and can be reused by calling `execute()`. See [code](https://github.com/datavisyn/tdp_core/blob/develop/tdp_core/db.py#L122)

Reply:
Yes, true. I did not see the difference in the `self.execute` call. Thanks for the clarification.